lib/processor/hosted/cpu.h

```diff
@@ -16,7 +16,7 @@
 
 static inline void metal_cpu_yield(void)
 {
-	metal_wait_usec(10);
+		k_busy_wait(10);
```

Comment:
This should be removed ?

Reply:
Or I think we should update metal_wait_usec with k_busy_wait for zephyr specific case.